### ISSUE TITLE
Add section about breaking APIs and PRs to upstream projects via godoc.org importers.

### ISSUE THREAD

**Issue by dmitshur** [shurcooL/SLA#1]
I should mention my policy, somewhat similar to [go4.org](https://github.com/camlistore/go4#details):

> * **no backwards compatibility**. go4 makes no backwards compatibility
>     promises. If you want to use go4, vendor it. And next time you
>     update your vendor tree, update to the latest API if things in go4
>     changed. The plan is to eventually provide tools to make this
>     easier.
> 
> * **forward progress** because we have no backwards compatibility,
>     it's always okay to change things to make things better. That also
>     means the bar for contributions is lower. We don't have to get the
>     API 100% correct in the first commit.

Except I don't necessarily recommend always vendoring. As long as the users of the repo are known via godoc.org importers, I will send upstream PRs to fix any code I break with my changes.

E.g., see https://github.com/bradfitz/go-issue-mirror/pull/5, https://github.com/bradfitz/go-issue-mirror/pull/6, and https://github.com/bradfitz/go-issue-mirror/pull/16 as some recent examples.

Basically, if you import my code into your code without vendoring, I consider myself responsible to help maintain your code. If you do vendor, then I'm not as responsible.

Also:

> * **no Go version policy** go4 packages are usually built and tested
>     with the latest Go stable version. However, go4 has no overarching
>     version policy; each package can declare its own set of supported
>     Go versions.

I should mention my Go version policy. It's to support latest version of Go only.
